README.md

```diff
@@ -6,109 +6,87 @@
+## [Click here to go straight to the Bootcamp Now!](https://github.com/freechipsproject/chisel-bootcamp)
 
-## Overview
-The standard Chisel3 compilation pipeline looks like:
-- Chisel3 (Scala) to Firrtl (this is your "Chisel RTL").
-- [Firrtl](https://github.com/ucb-bar/firrtl) to Verilog (which can then be passed into FPGA or ASIC tools).
-- Verilog to C++ for simulation and testing using [Verilator](http://www.veripool.org/wiki/verilator).
+---
 
-## Installation
-This will walk you through installing Chisel and its dependencies:
-- [sbt](http://www.scala-sbt.org/), which is the preferred Scala build system and what Chisel uses.
-- [Firrtl](https://github.com/ucb-bar/firrtl), which compiles Chisel's IR down to Verilog.
-  If you're building from a release branch of Chisel3, separate installation of Firrtl is no longer required: the required jar will be automatically downloaded by sbt.
-  If you're building chisel3 from the master branch, you'll need to follow the directions on the [Firrtl repository](https://github.com/ucb-bar/firrtl) to publish a local copy of the required jar.
-- [Verilator](http://www.veripool.org/wiki/verilator), which compiles Verilog down to C++ for simulation.
-  The included unit testing infrastructure uses this.
-
-### (Ubuntu-like) Linux
-
-1. Install Java
-   ```
-   sudo apt-get install default-jdk
-   ```
-1. [Install sbt](http://www.scala-sbt.org/release/docs/Installing-sbt-on-Linux.html),
-    which isn't available by default in the system package manager:
-    ```
-    echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
-    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 642AC823
-    sudo apt-get update
-    sudo apt-get install sbt
-    ```
-1. Install Verilator.
-    We currently recommend Verilator version 4.006.
-    Follow these instructions to compile it from source.
-
-    1. Install prerequisites (if not installed already):
-        ```
-        sudo apt-get install git make autoconf g++ flex bison
-        ```
-
-    2. Clone the Verilator repository:
-        ```
-        git clone http://git.veripool.org/git/verilator
-        ```
-
-    3. In the Verilator repository directory, check out a known good version:
-        ```
-        git pull
-        git checkout verilator_4_006
-        ```
-
-    4. In the Verilator repository directory, build and install:
-        ```
-        unset VERILATOR_ROOT # For bash, unsetenv for csh
-        autoconf # Create ./configure script
-        ./configure
-        make
-        sudo make install
-        ```
-
-### Arch Linux
+# Build circuits with Chisel
 
-```
-yaourt -S firrtl-git verilator sbt
-```
+When you are ready to build and test your own circuits. Chisel3 uses the Sonatype/Nexus/Maven package management systems to seamlessly deliver the environment and extension
```

Comment:
I think this is where you should link the installation instructions.

Also, if you're encouraging people to use the template, you don't need to go much into Sonatype/Nexus/Maven, since that's all abstracted away. But you might consider phrasing it like:

> When you are ready to build and test your own circuits, check out the [install instructions] for your platform. We recommend starting your project from the [chisel-template], though you can also [manually set up your project] to use the managed dependencies published on Sonatype/Nexus/Maven.